lib/WP_Auth0_Routes.php

```diff
@@ -98,10 +100,27 @@ public function custom_requests( $wp, $return = false ) {
 			return $output;
 		}
 
+		if ( $json_header ) {
```

Comment:
is this OK here or should it go before the `if ($return)` above ?? Maybe the `send_headers()` function might need to be called as well and you missed that (I don't really know)

Reply:
No need for a header if we're just spitting back the data rather than outputting.